### ISSUE TITLE
Different timeouts based on message level (info, warning, error)

### ISSUE THREAD

**Issue by fprevost** [CodeSeven/toastr#390]
It should be nice to define different timeouts depending on message level.
A warning or an error message should be visible longer than information one.

**Comment by TimFerrell** [comment]
You can set the timeout on each toastr individually.

Does that help?

**Comment by fprevost** [comment]
Yes I know, but I think that a default timeout by message level will be a good thing.
Infomation message can be unread without risk but warning or error one should be read and it should be interesting if they stay longer on screen.

**Comment by jonibatista** [comment]
I think this issue can be merged with #417.

**Comment by DRFrenette** [comment]
For anyone that comes across this, I was crudely able to accomplish this by creating a JavaScript class/wrapper for the notification, instantiating it with the type of notification, and then setting options via booleans with that.

`ToastrNotification = function(type, message) = function() {
  if (type === 'info') {
    toastr.options = { some stuff }
  } else {
    toastr.options = { some different stuff}
  }
}`

Then at some point: 

`new ToastrNotification('info', 'super important message');`
